genesis_core/user_api/iam/api/controllers.py

```diff
@@ -33,12 +37,49 @@
 from restalchemy.openapi import utils as oa_utils
 import pyotp
 
+from genesis_core.common import constants as common_c
 from genesis_core.user_api.iam.api import openapi_specs as oa_specs
 from genesis_core.user_api.iam.clients import idp
 from genesis_core.user_api.iam.dm import models
 from genesis_core.user_api.iam import constants as c
 from genesis_core.user_api.iam import exceptions as iam_e
 
+iam_cli_opts = [
+    cfg.BoolOpt(
+        "captcha_required",
+        default=False,
+        help="Check captcha",
+    ),
+    cfg.StrOpt(
+        "captcha_salt",
+        default=common_c.DEFAULT_CAPTCHA_SALT,
+        help="Captcha salt",
+    ),
+    cfg.StrOpt(
+        "captcha_key",
+        default=common_c.DEFAULT_CAPTCHA_KEY,
+        help="Captcha key",
+    ),
+    cfg.IntOpt(
+        "captcha_expires",
+        default=10,
+        help="Captcha expires minutes",
+    ),
+    cfg.IntOpt(
+        "captcha_max_number",
+        default=100000,
+        help="Captcha max number",
+    ),
+]
+
+DOMAIN = "iam_api"
+CONF = cfg.CONF
+
+CONF.register_cli_opts(
```

Comment:
Please move configs and their declarations in appropriate place https://github.com/infraguys/genesis_core/blob/master/genesis_core/cmd/user_api.py , controller module may use config but it should not register it.